### DISCUSSION TITLE
Live migration from ArgoCD

### DISCUSSION THREAD

**Discussion by rassie** [fluxcd/flux2#4938] [General]
In my environment I'm using both ArgoCD and Flux in different deployment stages, mostly for historical reasons. I'm slowly scheming to replace ArgoCD by Flux in the remaining stages, however, I wonder if it's possible to do so without service interruption, i.e. somehow make Flux "take over" resources managed by ArgoCD, so that I wouldn't need to export and re-import data etc. 

Has anyone done something like that? How complicated would this be? How would you tackle it?

Thanks!

**Comment by o6uoq** [comment]
FWIW: https://github.com/flux-subsystem-argo/flamingo

**Comment by stefanprodan** [comment]
Why do you need to export and re-import data? Flux can take over the reconciliation of any Kubernetes resource without replacing it.

**Reply by rassie** [reply]
That's more or less the question -- can it? Main problem being: if I remove something from Argo, normally it's gone immediately. Do I disable auto-prune in Argo and let Flux take over?

**Reply by stefanprodan** [reply]
Yes that should work
